src/components/Navbar.js

```diff
@@ -48,8 +62,12 @@ function Navbar() {
   const classes = useStyles();
   return (
     <AppBar className={classes.navContainer} color="primary" position="static">
-      <Link href="/">
-        <Image src={logo} width={180} height={30} />
+      <Link href="/" className={classes.logo}>
+        <Image
+          src={isMobileScreen ? iconLogo : logo}
+          width={isMobileScreen ? 24 : 180}
+          height={30}
+        />
```

Comment:
@VWRoli I'm thinking on this, how about <Hidden> vs {condition?r1:r2} ?
Maybe Hidden is more readable maintainable solution?

Reply:
Not sure what you mean here. 'isMobileScreen' should be 'Hidden'?